extensions/prost/private/prost.bzl

```diff
@@ -269,7 +269,7 @@ def _rust_prost_aspect_impl(target, ctx):
         crate = dep_variant_info.crate_info,
         cfgs = cfgs,
         env = dep_variant_info.crate_info.rustc_env,
-        deps = rust_analyzer_deps,
+        deps = depset([dep.id for dep in rust_analyzer_deps]).to_list(),
```

Comment:
Because CI wasn't added for prost, a regression from https://github.com/bazelbuild/rules_rust/pull/3028 was added. This should fix it if I understand the change correctly.